prebuilt-tasks/src/test/java/com/redhat/parodos/tasks/git/GitArchiveTest.java

```diff
@@ -0,0 +1,124 @@
+package com.redhat.parodos.tasks.git;
+
+import com.redhat.parodos.workflows.work.WorkContext;
+import com.redhat.parodos.workflows.work.WorkStatus;
+import junit.framework.TestCase;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.lib.Repository;
+import org.eclipse.jgit.storage.file.FileRepositoryBuilder;
+import org.junit.After;
+import org.junit.Test;
+import org.junit.jupiter.api.BeforeEach;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+public class GitArchiveTest extends TestCase {
```

Comment:
why `extends TestCase`?

Reply:
to be able to have assertNull, assertEquals to be tested? Is how idea generated it

Reply:
Ok, Already changed due to the comment from moti.